docs/claimantmodel/README.md

```diff
@@ -0,0 +1,113 @@
+# Sigstore Claimant Model
+
+Sigstore's claimant model includes a set of claims produced by Rekor and Fulcio logs. See [Claimant Model](https://github.com/google/trillian/blob/master/docs/claimantmodel/CoreModel.md) for more information and terminology.
+
+Sequence diagrams generated with [Claimant Model Render Tool](https://github.com/google/trillian/tree/master/docs/claimantmodel/experimental/cmd/render). This tool is a work in progress, so there may be some errors in the generated models.
+
+Each claim is in `model.md` in each folder. `full.md` contains the claim made in both the domains of Rekor or Fulcio and the log. The sequence diagram shows the interactions between all Actors. 
+
+All claims are also provided in this readme.
+
+## Fulcio: Identity-based signing
+
+<dl>
+<dt>Claim<sup>Fulcio</sup></dt>
+<dd><i>${OIDCIDOwner} authorizes ${CA} to bind ${PubKey} to ${OIDCIdentity}</i></dd>
+<dt>Statement<sup>Fulcio</sup></dt>
+<dd>X.509 certificate containing ${PubKey} and ${OIDCIdentity}, signed by ${CA}</dd>
+<dt>Claimant<sup>Fulcio</sup></dt>
+<dd>${CA}</dd>
+<dt>Believer<sup>Fulcio</sup></dt>
+<dd>Software Installer</dd>
+<dt>Verifier<sup>Fulcio</sup></dt>
+<dd>${OIDCIDOwner}: <i>${OIDCIDOwner} authorizes ${CA} to bind ${PubKey} to ${OIDCIdentity}</i></dd>
+<dt>Arbiter<sup>Fulcio</sup></dt>
+<dd>Community</dd>
+</dl>
+
+## Rekor: Identity-based signature
+
+<dl>
+<dt>Claim<sup>Rekor</sup></dt>
+<dd><i>${OIDCIdentity} signs ${Hash} using the key bound by ${Certificate}</i></dd>
+<dt>Statement<sup>Rekor</sup></dt>
+<dd>${Hash}, X.509 certificate ${Certificate} containing ${OIDCIdentity}, and signature over ${Hash}</dd>
+<dt>Claimant<sup>Rekor</sup></dt>
+<dd>${OIDCIdentity}</dd>
+<dt>Believer<sup>Rekor</sup></dt>
+<dd>Software Installer</dd>
+<dt>Verifier<sup>Rekor</sup></dt>
+<dd>${OIDCIDOwner}: <i>${OIDCIdentity} signs ${Hash} using the key bound by ${Certificate}</i></dd>
+<dt>Arbiter<sup>Rekor</sup></dt>
+<dd>Community, identity-artifact mapping</dd>
+</dl>
+
+## Rekor: Key-based signature
+
+<dl>
+<dt>Claim<sup>Rekor</sup></dt>
+<dd><i>${Key} signs ${Hash}, verifiable with ${PubKey} </i></dd>
+<dt>Statement<sup>Rekor</sup></dt>
+<dd>${Hash}, public key ${PubKey}, and signature over ${Hash}</dd>
+<dt>Claimant<sup>Rekor</sup></dt>
+<dd>${Key}</dd>
+<dt>Believer<sup>Rekor</sup></dt>
+<dd>Software Installer</dd>
+<dt>Verifier<sup>Rekor</sup></dt>
+<dd>${KeyOwner}: <i>${Key} signs ${Hash}, verifiable with ${PubKey}</i></dd>
+<dt>Arbiter<sup>Rekor</sup></dt>
+<dd>Community, key-artifact mapping</dd>
+</dl>
+
+## Rekor: Provenance
```

Comment:
There's also a Provenance version with `${PubKey}` instead of `X.509 certificate ${Certificate} containing ${OIDCIdentity}`, correct?

(Of course, this somewhat introduces the problem of distributing `${PubKey}`, but `${PubKey}` could be a long-lived CA-signed certificate.)

I don't know if you need to add the whole section a second time, but maybe a note that this configuration is possible (assuming that I know what I'm talking about, which is suspect...).

Reply:
Yep, that is a supported configuration, but its claim will look very similar to the existing key-based claim. FWIW, there are many supported identities beyond just x509 certs and public keys too, this covers the primary use cases.